README.rst

```diff
@@ -317,7 +317,7 @@ The plugin implementation will look something like the example below. This is
 a simplified example based on a working plugin implementation.
 
 .. code:: python
-    import diff_cover
+    import diff_cover.hook import hookimpl as diff_cover_hookimpl
```

Comment:
This is not quite right. I tested on the SQLFluff plugin, and this is what we need:
```
from diff_cover.hook import hookimpl as diff_cover_hookimpl
```

Reply:
Thanks for catching this!